api/api-doc/storage_service.json

```diff
@@ -2495,7 +2495,23 @@
                ]
             }
          ]
-      }      
+      },
+      {
+         "path":"/storage_service/raft_topology_test/reload",
+         "operations":[
```

Comment:
I guess this will be useful to rescue clusters in a bad situation too. Maybe we should change the API not to have test in it, but add more explanations (and warnings) about when not to use it.

Reply:
In theory, it should be always safe to do this reload, it doesn't do anything if we don't modify system.topology (which the user should never do, unless they are indeed rescuing the cluster in maintenance mode or something).

I can remove the `_test` part from the name.

Reply:
> it doesn't do anything 

well to be precise, it does load data from disk and construct the in-memory topology data structure, but it should be the same as before.

Reply:
cc @gleb-cloudius

Reply:
I do not see any problem with forcing reload I just hope we will not have bugs that will require it.